docs/2101UserGuide.md

```diff
@@ -62,50 +62,56 @@ CCACommander Ultra Promax Xtra 9000PLUS is the one-stop app for CCA Heads to man
+`createMember n/Lim Jun Jie g/Male p/83498018 e/limjunjie@gmail.com a/19 Kent Ridge Crescent, Singapore 119278 t/publicity`
+
+<figure>
+    <img src="images/createMemberPostCommand.png"
+         alt="createMember Post Command">
+    <figcaption>After executing the <code>createMember</code> command</figcaption>
+</figure>
 
 <hr class="command-separator">
 
-#### Delete a Member : `deleteMember`
+#### Edit a Member: `editMember`
+If you accidentally make a typo while creating your member, do not worry! You can quickly fix it with the `editMember` command.
 
-Deletes the member at the specified index.
+Here is how you can use it:<br>
+`editMember MEMBER_INDEX [n/MEMBER_NAME] [g/GENDER] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
 
-Format: `deleteMember MEMBER_INDEX`
+<div markdown="block" class="alert alert-primary">
+:bulb: At least one field must be provided
+</div>
 
-* Deletes the member at the specified `MEMBER_INDEX`.
-* The index refers to the index number shown in the **currently displayed** member list.
-* The index **must be a positive integer** that is within the range of the length of the member list.
+Here is an example command where we edit the first **member** in the displayed list, changing their **phone number** to "91234567" and **email** to "johndoe@example.com":
 
-Examples:
-* `deleteMember 1` deletes the 1st member in the member list.
-* `deleteMember 10` deletes the 10th member in the member list.
+`editMember 1 p/91234567 e/johndoe@example.com`
 
 <hr class="command-separator">
+#### Delete a Member : `deleteMember`
 
-#### Edit a Member: `editMember`
-Edits the member at the specified index with the specified fields.
+If a member leaves your CCA, get over the disappointment by swiftly removing them from CCACommander using the `deleteMember` command!
 
-Format: `editMember MEMBER_INDEX [n/MEMBER_NAME] [g/GENDER] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+Here is how you can use it:<br>
+ `deleteMember MEMBER_INDEX`
 
-* The index refers to the index number shown in the **currently displayed** member list.
-* The index **must be a positive integer** that is within the range of the length of the member list.
-* At least one field to edit must be provided.
-* Acceptable values for `GENDER`: `Male`, `Female`, `Others`.
-* Acceptable values for `EMAIL`: A string with an email extension (e.g. `@gmail.com`).
+Here is an example command where we delete the first **member** in the displayed list:
 
-Examples:
-* `editMember 1 a/RH t/Musician` edits the address and the tag fields of the 1st member in the member list.
+`deleteMember 1`
+
+<div markdown="block" class="alert alert-primary">
+:bulb: The `MEMBER_INDEX` parameter refers to the index number shown in the **currently displayed** member list. We recommend you to use the `list` command to see the full member list before using the `editMember` or `deleteMember` commands.
```

Comment:
I think this tip can be changed to an information block and the recommendation removed, because there are still use cases as to why a user may want to edit or delete a member without using the `list` command, for example after a find.

Reply:
Agree, will do that